src/components/Input/Input.stories.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import Input from './Input';
+
+const StatefulInput = props => {
+  const [value, setValue] = useState(props.initialValue);
+
+  const handleChange = event => {
+    action('onChange')(event);
+    setValue(event.target.value);
+  };
+
+  const handleFocus = event => {
+    action('onFocus')(event);
+  };
+
+  const handleBlur = event => {
+    action('onBlur')(event);
+  };
+
+  return (
+    <Input
+      value={value}
+      onChange={handleChange}
+      onFocus={handleFocus}
+      onBlur={handleBlur}
+      { ...props }
+    />
+  );
+};
+
+storiesOf('Input', module)
+  .add('Basic Text Input', () => {
+    const basicInputValue = 'Hello World!';
+    const disabledInputValue = 'I am disabled';
+    const placeholderInputValue = '';
+    const withLabelInputValue = 'I have a label!';
+
+    return (
+      <>
+        <div style={{ marginBottom: '1rem' }}>
```

Comment:
should we make this div of `{ marginBottom: '1rem' }` a reusable component?? something like a `StoryComponentContainer`

Reply:
For sure! I kinda slapped this together to get a story going but as we hadd more stories it will be beneficial to figure out the best way to display these. We also will have access to utility classes from our token library so in the future we might just do something like `<div className="m-b-sm">` (margin-bottom-small) but we're in the process of building all these libraries right now.

Reply:
:+1: